recipes/jep/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jep" %}
+{% set version = "3.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7a15b6279b703c8af6b1f002186c542770f9059e8b3fe724914f664e67b7e595
+  patches:
+    - fix-split.patch
+    - fix-overlinking.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  ignore_run_exports:
+    - openjdk
+    - numpy
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - openjdk=8
+    - pip
+    - python
+    - numpy
+  run:
+    - openjdk
+    - python
+    - {{ pin_compatible('numpy') }}
+
+about:
+  home: "https://github.com/ninia/jep"
+  license: "zlib/libpng"
```

Comment:
Is this the license https://spdx.org/licenses/zlib-acknowledgement.html?

If yes, then use this name for it

```
zlib-acknowledgement
```